test/integration/targets/iosxr_lacp_interfaces/tests/cli/replaced.yaml

```diff
@@ -0,0 +1,51 @@
+---
+- debug:
+      msg: "START iosxr_lacp_interfaces replaced integration tests on connection={{ ansible_connection }}"
+
+- include_tasks: _remove_config.yaml
+
+- include_tasks: _populate.yaml
+
+- block:
+    - name: Replace device configurations of listed interfaces with provided configurations
+      iosxr_lacp_interfaces: &replaced
+        config:
+          - name: Bundle-Ether10
+            churn_logging: partner
+
+          - name: GigabitEthernet0/0/0/1
+            period: 300
+        state: replaced
+      register: result
+
+    - name: Assert that correct set of commands were generated
+      assert:
+        that:
+          - "{{ replaced['commands'] | symmetric_difference(result['commands']) |length == 0 }}"
+
+    - name: Assert that before dicts are correctly generated
+      assert:
+        that:
+          - "{{ populate | symmetric_difference(result['before']) |length == 0 }}"
+
+    - name: Assert that after dict is correctly generated
+      assert:
+        that:
+          - "{{ replaced['after'] | symmetric_difference(result['after']) |length == 0 }}"
+
+    - name: Replace device configurations of listed interfaces with provided configurarions (IDEMPOTENT)
+      iosxr_lacp_interfaces: *replaced
+      register: result
+
+    - name: Assert that task was idempotent
+      assert:
+       that:
+         - "result['changed'] == false"
```

Comment:
Same like above.

Reply:
If result.changed would not be false if commands list is not empty.